setup-dev-env.sh

```diff
@@ -82,13 +84,17 @@ if [ "$option_no_cuda_drivers" = "true" ]; then
 fi
 
 # Check installation of dev package
-if [ "$option_no_dev" = "true" ]; then
+if [ "$option_runtime" = "true" ]; then
     ansible_args+=("--extra-vars" "install_devel=false")
-    ansible_args+=("--extra-vars" "installation_type=ros-base")
```

Comment:
@caiyuexiang Why do you want to split this option out of this if-statement?
I'm asking this because we need to keep the decision logs for future reviews.

Reply:
Hi, @kenji-miyake , split this option just to maintain a uniform style of context, each option controls one args. And those two parameters under --runtime are added aims to minimize the options for exposed interface.

Reply:
@caiyuexiang Thank you, but I'm sorry, I don't understand well. :cry: 
The if-statement was split, but both options are set simultaneously when `--runtime` is given.
So I don't understand the reason for splitting the if-statement.
Could you elaborate on this a bit more?

Reply:
Hi, @kenji-miyake Since it's easy to cause confusion here, maybe I should keep the original if-statement and just replace the  --no-dev option with --runtime. What do you think?

Reply:
It sounds clear and good to me!

Reply:
If you want to change something, it's of course okay, but the reason should always be stated and reviewed.